xc7/utils/prjxray_routing_import.py

```diff
@@ -230,6 +231,28 @@ def check_feature(feature):
 
         return ' '.join((feature, enable_buffer_feature))
 
+    if CLK_HROW_CK_MUX_REGEX.fullmatch(feature_path[-1]):
```

Comment:
Let's add a comment in words what is going on here.

Reply:
Done